authentication-request-handler/src/main/java/org/wso2/carbon/identity/framework/extended/ExtendedAuthenticationRequestHandler.java

```diff
@@ -0,0 +1,98 @@
+package org.wso2.carbon.identity.framework.extended;
```

Comment:
Please add the license header.

Reply:
Fixed with c907fcc2a908e093c988f8bf86c54ce8e98b72c0